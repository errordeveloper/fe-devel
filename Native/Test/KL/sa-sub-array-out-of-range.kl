/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

operator entry() {
  String slicedArray<>( Size(16) );
  for ( Size i=0; i<16; ++i )
    slicedArray[i] = String(i);
  String subSlicedArray<>( slicedArray, Size(4), Size(8) );
  report(subSlicedArray[3]);
  report(subSlicedArray[8]);
}
