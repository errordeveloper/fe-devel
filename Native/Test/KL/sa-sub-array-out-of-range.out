7
Caught exception: sa-sub-array-out-of-range.kl:11:10: index (8) out of range (8)
